rest-api-spec/src/main/resources/rest-api-spec/api/indices.delete.json

```diff
@@ -20,6 +20,11 @@
         "master_timeout": {
           "type" : "time",
           "description" : "Specify timeout for connection to master"
+        },
+        "include_hidden": {
```

Comment:
we have lots of them without `expand_wildcards` where they should be so I didn't add them